WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -421,6 +440,8 @@ public void afterTextChanged(Editable s) {
     public void onResume() {
         super.onResume();
         ActivityId.trackLastActivity(ActivityId.COMMENT_DETAIL);
+        AnalyticsUtils.trackCommentActionWithSiteDetails(
+                Stat.COMMENT_VIEWED, mCommentSource.toAnalyticsCommentActionSource(), mSite);
```

Comment:
maybe we should avoid tracking this if we are coming back from another screen? wdyt?

Reply:
This was a bit tricky, but I think I made it work 👍